cli/execute.py

```diff
@@ -52,6 +28,10 @@ def ansible_playbook(playbook, args, inventory="local_hosts"):
      :args: all arguments passed for the playbook
      :inventory: the inventory file to use, default: local_hosts
     """
+    verbosity = args["verbose"]
+    display = Display(verbosity=verbosity)
+    import __main__ as main
+    setattr(main, "display", display)
```

Comment:
Why do we need that? If 'display' is required for some one else, then maybe it should be passed to the method as input argument or configured in the 'config' module